src/Extracting/Strategies/GetFromInlineValidatorBase.php

```diff
@@ -51,7 +51,7 @@ public function lookForInlineValidationRules(ClassMethod $methodAst): array
                 break;
             } else if (
                 // Try $validator = Validator::make(...)
-                $rvalue instanceof Node\Expr\StaticCall && end($rvalue->class->parts) == "Validator"
+                $rvalue instanceof Node\Expr\StaticCall && end($rvalue->class->parts ?: []) == "Validator"
```

Comment:
Ah no, yours was better.

```suggestion
                $rvalue instanceof Node\Expr\StaticCall && $rvalue->class->parts && end($rvalue->class->parts) == "Validator"
```